dotcom-rendering/src/web/lib/braze/taylorReport.ts

```diff
@@ -0,0 +1,7 @@
+import { TagType } from '../../../types/tag';
+
+const tagId = 'news/series/cotton-capital';
+
+export const suppressForTaylorReport = (tags: TagType[]): boolean =>
```

Comment:
Tiny nitpick - should this be `suppressForNotTaylorReport` or similar? We'll only suppress on non-Taylor Report articles.

Reply:
good point! Naming is hard :)